package.bzl

```diff
@@ -0,0 +1,45 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
nit: I think we should use the current year in new files?

Reply:
👍